inc/passwords/namespace.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+namespace HM\Platform\Security\Passwords;
+
+use function HM\Platform\get_config;
+use WP_Error;
+use ZxcvbnPhp\Zxcvbn;
+
+/**
+ * Bootstrap.
+ */
+function bootstrap() {
+	add_action( 'check_passwords', __NAMESPACE__ . '\\enforce_password_strength', 10, 3 );
+	add_action( 'admin_print_styles-user-edit.php', __NAMESPACE__ . '\\hide_weak_password_prompt' );
+	add_action( 'admin_print_styles-profile.php', __NAMESPACE__ . '\\hide_weak_password_prompt' );
+}
+
+/**
+ * Get minimum required strength for passwords.
+ *
+ * @return int
```

Comment:
Needs documentation, probably to be figured out how this overlaps or not with the filter doc block below. Probably just needs to be duplicated.